cluster/aws/util.sh

```diff
@@ -1342,7 +1494,9 @@ function wait-minions {
 
 # Wait for the master to be started
 function wait-master() {
-  detect-master > $LOG
+  if [[ "${PRESET_INSTANCES_ENABLED:-}" != "yes" ]]; then
```

Comment:
I saw string "yes" scattered everywhere. Can we use constant for "yes"? Just try to avoid lowercase/uppercase issue.

Reply:
Updated.